client/components/jetpack/daily-backup-status/status-card/no-backups-yet.jsx

```diff
@@ -45,13 +45,10 @@ const NoBackupsYet = () => {
 			<div className="status-card__label">
 				{ isJetpackCloud()
 					? translate(
-							'Your first backup will appear here {{strong}}within 24 hours{{/strong}} and you will receive a {{wpcomLink/}} notification once the backup has been completed.',
+							'Your first backup will appear here {{strong}}within 24 hours{{/strong}} and you will receive an email once the backup has been completed.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Your first backup will appear here {{strong}}within 24 hours{{/strong}} and you will receive a notification once the backup has been completed.' )` *ES Score: 9*

Reply:
Do we want to keep this verbiage, or would it cause confusion?

Reply:
There is an asana ticket that references this verbiage and suggests a small change here: 1164141197617539-as-1196090710229513 , but not totally sure what the verbiage should be exactly.

Reply:
@elliottprogrammer - I think we can handle that in that ticket, especially as there's no verbiage there yet.